src/content/protocols/spark.md

```diff
@@ -0,0 +1,414 @@
+
+Oracles have a killswitch to pause all borrowing if an asset's price falls below a predetermined threshold. If the killswitch is triggered (see [dependencies](#dependencies)), the borrowing is disabled without delay for all assets across the protocol until the killswitch is reset by the governance.
+
+> Exit Window Score: High
+
+## Accessibility
+
+Spark has a main open-source frontend at [spark.fi](https://app.spark.fi). The code is open-source and there are instruction to run the page locally as a developer. There are no dedicated instructions for self-hosting. Direct interaction with contract is not straight forward nor documented.
+
+However, SparkLend is accessible on third-party applications such as DeFiSaver and SummerFi. `sUSDS` (`USDS` in savings) can also be swapped back to `USDS` and `USDC` using common decentralized exchanges. These apps build an acceptable backup solution in case of failure of the official frontend and backend.
+
+> Accesibility score: Low
+
+# Technical Analysis
+
+## Spark ALM
+
+Spark has a liquidity management layer (ALM) whose role is to automate liquidity provision of `USDS`, `sUSDS`, and `USDC` directly from Sky and across blockchain networks and DeFi protocols. The overview of this system is shown in the diagram below. All core permissions are held by the Sky Governance through its delayed proxy (`DSPause Proxy`). The `Spark Proxy` is a simple wrap around this governance to distinguish permissions specific to spark. The `AllocatorVault` bridges between Sky and Spark and allows minting/burning of `USDS`.
+
+The main component of the ALM is the `ALMProxy` and its `MainnetController`. The `ALMProxy` holds all the funds while the `MainnetController` can move and distribute those funds accross protocols and chains. Off-chain _Relayers_ can dictate all liquidity movements but are subject to strict rate limits. A _Freezer_ multisig can freeze the _Relayers_ at any time if it detects suspicious behaviors. The `MainnetController` supports the provision of liquidity in Aave, Ethena, and any ERC-4626 compatible vault.
+
+![Overview of the Spark's liquidity layer](./diagrams/spark-ALM.png)
+
+## SparkLend
+
+SparkLend is a fork of Aave v3 with some particularities highlighted in the diagram below. Similarly to [Spark' liquidity layer](#spark-alm) all core permissions are held by the Sky Governance through the `DSPause Proxy`. Some emergency actions can be taken such as freezing all new borrowing in SparkLend if a certain asset price falls below a predetermined threshold. A multisig called _SparkLend Freezer_ as well as the Sky governance can freeze or pause any market in SparkLend without any delay. They can then be reactivated through the delayed governance. The core contracts in SparkLend are non-upgradeable, but the treasuries as well as the rewards contract may be upgraded. Upgrading the rewards may lead to the loss of unclaimed yield, as we consider rewards to be yield as well.
```

Comment:
what are "core permissions"? maybe better write "permissions to upgrade and other critical permissions" or so

Reply:
mention the following somewhere further up in the paragraph (right after explaining what SparkLend is and that it is an Aave fork), as this is a key information: "Most contracts are non-upgradeable with upgrades only possible on the Treasury and RewardsControllerManager contracts."

Reply:
maybe structure this section a bit more along the following paragraphs:
- what is the intended purpose of the SparkLend module
- SparkLend is an aave fork but with the following key differences in terms of configuration (eg also only limited scope of markets)
- are contracts upgradeable? what are the remaining permissions and the impact for users (loss of funds, loss of unclaimed yield, material change in expected protocol performance)